Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/WebView/Compatible/WebViewCompatibilityAdapter.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Windows;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
+{
+    internal sealed class WebViewCompatibilityAdapter : WebBaseCompatibilityAdapter
+    {
+        private WebView _webView = new WebView();
+
+        public override Uri Source
+        {
+            get => _webView?.Source;
```

Comment:
Why the null check if the field is always created during construction of the type? Other properties are not guarded the same way